### PR TITLE
fix deprecated for php8.1 + upgrade to symfony 6.0

### PR DESCRIPTION
**Pull request by jgroc-de**
in composer:
- adding php8.1 compatibility
- adding symfony 6.0 compatibility

fix in code:
- adding return typehint (as requested by sf6 + php8.1)
- in test sf config: setting utf8: true (as it will be set to true anyway by sf6

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,21 +5,21 @@
     "license": "MIT",
     "type": "symfony-bundle",
     "require": {
-        "php": "^7.4 || ^8.0",
-        "symfony/framework-bundle": "^4.4 || ^5.0",
+        "php": "^7.4 || ^8.0 || ^8.1",
```
Comment (goetas) [inline composer.json]: `^8.0`  is equivalent to `^8.1`

Comment (jgroc-de) [reply composer.json]: removed (thx for the comment)

**Comment by arnedesmedt** [comment]
Hi, Can this one be reviewed again? Thx in advance

**Review by goetas — APPROVED** [review]

**Comment by jgroc-de** [comment]
Hi, thx for approving this. What is the next step? Can you merge it?

**Comment by styxiak** [comment]
Hello,
Is this possible to merge this?

**Comment by chrisguitarguy** [comment]
sorry, y'all, I will look into this hopefully by the end of this weekend (2022-04-10). Looks like I need to figure out another CI solution since TravisCI changed.

**Comment by chrisguitarguy** [comment]
sorry, to just close this. I had to do make some changes to get github actions running and created some conflicts here. Didn't want y'all to have to do some run around due to, essentially, house keeping.

https://github.com/chrisguitarguy/RequestIdBundle/pull/25 has Symfony 6.X support and I'll tag a v5.0.0-RC1 release now for y'all to try out. I'll set a reminder for myself two weeks from now to do a a v5.0.0 release if no one tells me otherwise here. I'm not on Symfony 6 myself yet.

**Pull request closed without merging**